### DISCUSSION TITLE
Azure support ETA?

### DISCUSSION THREAD

**Discussion by maxmumford** [ArcadeData/arcadedb#919] [Q&A] [answered]
Hello,

I have emailed Arcade a couple of times about this but received no reply which I have to say didn't inspire much confidence from the outset!

We are looking for an alternative to Cosmos which must support Azure PaaS and gremlin. I have read in the ArcadeDB manual that Azure is coming soon™️. Do you have a roadmap or date for this?

Thanks

**Comment by lvca** [comment]
Hello @maxmumford, I didn't see your email. What address did you use?

We're still behind with our schedule on releasing ArcadeDB with an official image on both Amazon and Azure marketplaces. This is a community driver project, so any help on creating such image is very welcomed!

**Comment by maxmumford** [comment, marked as answer]
Hi @lvca. Thanks for the update. Do you have a rough ETA or is it too difficult to say at this point?

**Reply by lvca** [reply]
Hard to say right now. It's possible that tomorrow somebody gets in touch with us and start it, or perhaps a sponsor willing to pay for this feature, but no ETA, sorry.

**Reply by maxmumford** [reply]
OK no problem. Unfortunately we don't have the resources to contribute right now but we will certainly keep an eye out and if and when it comes out we will consider moving over from Cosmos.

Thank you

**Reply by robfrank** [reply]
Hi, I'm just curious, what do you need to "support" Azure? 
ArcadeDB is provided as a Docker image, is not enough?

**Reply by maxmumford** [reply]
Mostly we don't want to be managing our own VMs in azure. We want a fully managed service that includes things like backup and restore etc.
